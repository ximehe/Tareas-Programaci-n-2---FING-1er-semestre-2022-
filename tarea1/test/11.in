# implementar _rep_cadena, crearCadena, cantidadEnCadena, estaEnCadena
cantidadEnCadena
estaEnCadena 5
Fin