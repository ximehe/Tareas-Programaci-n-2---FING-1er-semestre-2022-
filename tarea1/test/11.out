1>#  implementar _rep_cadena, crearCadena, cantidadEnCadena, estaEnCadena.
2>La cantidad es 0.
3>5 no estÃ¡ en la cadena.
4>Fin.
